trade-finance-manager-api/src/v1/controllers/deal.party-db.test.js

```diff
@@ -53,3 +54,60 @@ describe('getCompany returns company', () => {
     expect(result.data).toEqual(mockCompany);
   });
 });
+
+jest.mock('../api.js', () => ({
```

Comment:
Instead of mocking should make actual calls to the API for a realistic API tests?